functions.lua

```diff
@@ -81,7 +81,7 @@ function wrench.pickup_node(pos, player)
 	if def.timer then
 		local timer = minetest.get_node_timer(pos)
 		local timeout = timer:get_timeout()
-		if timeout > 0 then
+		if timeout then
```

Comment:
I'm curious about why you changed this, `get_timeout()` always returns a value, with 0 being no timer running.

Is it different in other minetest versions? (I use 5.5.0-dev)